tools/apiview/parsers/apiview-treestyle-parser-schema/models.tsp

```diff
@@ -0,0 +1,150 @@
+model CodeFile {
+  /** The version of the parser */
+  VersionString: string;
+
+  /** Name the ocde file. Usually just the package name and version*/
+  Name: string;
+  
+  Language: string;
+  LanguageVariant: "Default" | "Spring" | "Android";
```

Comment:
Shouldn't this be optional, and the server assume `Default` if not present? If yes, you can write it like that.
```suggestion
  LanguageVariant?: "Default" | "Spring" | "Android" = "Default";
```

Reply:
LanguageVariant must be optional. As far as I understand, it's set by Java parser only.